_pages/security-events.md

```diff
@@ -223,8 +223,8 @@ To configure your application to receive notifications from Login.gov, supply Lo
 
 #### Production environment:
 
-- Open a Zendesk ticket to have your SET configuration added to our outbound proxy
-- The `push_notification_url` will be deployed as part of the application promotion process
+- The `push_notification_url` will be deployed as part of the application promotion process. Please specify in the
+promotion request that you are including a SET configuration so tht we know to add your URL to the outbound proxy.
```

Comment:
small typo

```suggestion
promotion request that you are including a SET configuration so that we know to add your URL to the outbound proxy.
```